sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/AlterTableRenamePartitionSuite.scala

```diff
@@ -17,16 +17,13 @@
 
 package org.apache.spark.sql.execution.command.v2
 
-import org.scalatest.Ignore
-
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.execution.command
 
 /**
  * The class contains tests for the `ALTER TABLE .. RENAME PARTITION` command
  * to check V2 table catalogs.
  */
-@Ignore
```

Comment:
Shall we do this separately? Maybe, we can do this first if this was due to some old issues.

Reply:
This is because https://github.com/apache/spark/pull/41533#discussion_r1226727363 is also to avoid similar issues. I think it should not be the root cause, so we can remove ignore. I agree to remove ignore in an independent pr. let me revert this one first

Reply:
test remove `@ignore` in https://github.com/apache/spark/pull/41647